### ISSUE TITLE
 ./autogen.sh: aclocal: not found

### ISSUE THREAD

**Issue by clarkk** [tesseract-ocr/tesseract#833]
```
cd /var/bin && wget https://github.com/tesseract-ocr/tesseract/archive/3.05.00.tar.gz -O tesseract-3.05.00.tar.gz && tar -xvf tesseract-3.05.00.tar.gz
cd tesseract-3.05.00 && ./autogen.sh
```

# error
```
# ./autogen.sh
Running aclocal
./autogen.sh: 69: ./autogen.sh: aclocal: not found

  Something went wrong, bailing out!
```

**Comment by Shreeshrii** [comment]
Try with the latest source from 3.05 branch.

Also check that you have all dependencies listed on compiling page.

- excuse the brevity, sent from mobile


On 22-Apr-2017 7:27 PM, "clarkk" <notifications@github.com> wrote:

cd /var/bin && wget
https://github.com/tesseract-ocr/tesseract/archive/3.05.00.tar.gz -O
tesseract-3.05.00.tar.gz && tar -xvf tesseract-3.05.00.tar.gz
cd tesseract-3.05.00 && ./autogen.sh

error

# ./autogen.sh
Running aclocal
./autogen.sh: 69: ./autogen.sh: aclocal: not found

  Something went wrong, bailing out!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/tesseract-ocr/tesseract/issues/833>, or mute the thread

**Comment by Shreeshrii** [comment]
https://github.com/tesseract-ocr/tesseract/wiki/Compiling

- excuse the brevity, sent from mobile

**Comment by amitdo** [comment]
>check that you have all dependencies listed on compiling page.
>https://github.com/tesseract-ocr/tesseract/wiki/Compiling 

Yes. The message means you didn't install automake.

**Comment by amitdo** [comment]
@zdenop, please push the button ... :)

**Comment by jetma** [comment]
```
yum install automake libtool -y
```

**Comment by TuulaP** [comment]
Thanks for above tips, in clean ubuntu had to install , to get autogen.sh to complete.

> sudo apt-get install automake libtool

**Comment by Oldlexar** [comment]
> ```
> yum install automake libtool -y
> ```

thanks dude

**Comment by lalmiAhmed** [comment]
thank you guys the problem solved on ubuntu 20.04 by : 
sudo apt install automake libtool
